packages/tupaia-web-server/src/routes/ProjectCountryAccessListRoute.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import { Route } from '@tupaia/server-boilerplate';
+import { TupaiaWebProjectCountryAccessListRequest } from '@tupaia/types';
+
+export type ProjectCountryAccessListRequest = Request<
+  TupaiaWebProjectCountryAccessListRequest.Params,
+  TupaiaWebProjectCountryAccessListRequest.ResBody,
+  TupaiaWebProjectCountryAccessListRequest.ReqBody,
+  TupaiaWebProjectCountryAccessListRequest.ReqQuery
+>;
+
+export class ProjectCountryAccessListRoute extends Route<ProjectCountryAccessListRequest> {
+  public async buildResponse() {
+    const { ctx, params } = this.req;
+    const { projectCode } = params;
+    const { projects } = await ctx.services.webConfig.fetchProjects();
+
+    const project = projects.find(({ code }: { code: string }) => code === projectCode);
+    if (!project) {
+      throw new Error(`No project found with code '${projectCode}'`);
+    }
+    const { names } = project;
+    const countryAccessList = await ctx.services.central.fetchResources('me/countries');
```

Comment:
It might be nice if we moved this fetch of `me/countries` to the `CentralApi` and wrapped it, perhaps something like:
```
  public async getCountryAccessList(): Promise<CountryAccess[]> {
    return this.connection.get('me/countries');
  }
```
That way we'd have the return type set in one place.

I don't mind that the other `CentralApi` routes return `any`, since typically we can cast them using the types from the `@tupaia/types` package. eg.
```
import { Entity }  from `@tupaia/types';

const countries: Entity[] = await central.fetchResources('entities', { filter: { type: 'country' } });
```

But in the `me/countries` instance we don't have a easily reference-able type def